docs/source/topics/proc_setting-up-remote-server.adoc

```diff
@@ -63,49 +63,28 @@ $ sudo cp /etc/haproxy/haproxy.cfg{,.bak}
 $ export CRC_IP=$({bin} ip)
 $ sudo tee /etc/haproxy/haproxy.cfg &>/dev/null <<EOF
 global
-    debug
+    log /dev/log local0
```

Comment:
As per https://www.haproxy.com/blog/the-four-essential-sections-of-an-haproxy-configuration/ (section log) 
```
Note that in order to read the logs, you will need to configure any of the syslog daemons, or journald, to write them to a file.
```
I am not sure if that need manual configuration or just these daemons suppose to present

Reply:
I'm also relatively confused as to how haproxy logging works :) These days, most systems have journald configured, so should be fine.

Reply:
@cfergeau I checked if we add `debug` flag then logs are part of journalctl.